### DISCUSSION TITLE
Welcome to Steamtools Discussions!

### DISCUSSION THREAD

**Discussion by st2024** [st2024/Steamtools#12] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by DogancanYr** [comment]
Will the software be translated into English ?

**Reply by st2024** [reply]
Software supports switching to English

**Reply by DogancanYr** [reply]
Could you show me where to do it?

**Reply by st2024** [reply]
Tray icon in the lower right corner

**Reply by DogancanYr** [reply]
How did I miss it.
 Thanks .

**Comment by DogancanYr** [comment]
How can I remove the games I added from the Steam library ?

**Reply by DogancanYr** [reply]
Ctrl+drag  ...

**Comment by bizdatech** [comment]
anyway thanks to you author, now can easily share with family members.

so far got any way to download full installation files for it? 

as i notice your example for RE village. i drag to there but of cause it won't let me download the files. 

unless i get it from else where?

**Reply by DogancanYr** [reply]
r/piratedgames,r/piracy, r/freemedıaheckyeah read megathreads and there is a guide in freemediaheckyeah mehathread.

**Reply by bizdatech** [reply]
thread for this  using steamtools?

before i meeting this was introduce by the other tools, google so hard cant find this steamtools until i some other friend intro. then i only meeting this tool which is working and easy to use.


but still no 1 guide how to do.

after knowing the method above. if i wan play online need someone share the original files out right? 

or fullfiles to play offline but only must steam folder only.

**Comment by bizdatech** [comment]
what the new version release log?

**Comment by WaifunMan** [comment]
Is it normal for Windows Defender to alert you of a Trojan?

**Reply by bizdatech** [reply]
Is it fine. As they work in background uncommon...

As long as we download from trusted developer..

But new version seems like got block by anti malware .


Anyway how to update the files.
Approved it try to update it still keep asking new update haha.

Havent try to delete and redo.

I scare the game I can play and setting will all gone.

**Reply by DogancanYr** [reply]
Read r/piracy r/piratedgames r/freemediaaheckyeah megathreads. 

This is only for cracking the game, it cannot do anything with the game files.

Update is usually done by deleting the old game and installing a new one. Check the save file location on Steamdb there should be no save file in the game file.
Or by finding the update file from Ovagames or the no installer torrent from cs.rin.ru

**Reply by bizdatech** [reply]
huh?

**Reply by DogancanYr** [reply]
At first I thought it was for steamtools, then for any game.

**Reply by bizdatech** [reply]
hmm sorry for my english. 

but i trying to say is the steamtools keep asking for updating even seems like already updated ,

first was block by antivirus . 

so i approved proceed for the injection of the apps.

then it still keep asking for updating. so i wondering if i delete the files and reinstall again, will be the setting all gone
